### DISCUSSION TITLE
Zoom meeting recording

### DISCUSSION THREAD

**Discussion by jpantoudis** [aclap-dev/video-downloadhelper#1406] [Help and Q&A]
### Your question

When i download a Zoom meeting recording that has live subtitles the subtitles do not download's 

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"02947c81e5bf4371"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0

**Comment by jcfrog** [comment]
Sorry, subtitles are not managed by VDH.
